x-pack/plugins/apm/kibana.json

```diff
@@ -22,17 +23,13 @@
   ],
   "server": true,
   "ui": true,
-  "configPath": [
-    "xpack",
-    "apm"
-  ],
-  "extraPublicDirs": [
-    "public/style/variables"
-  ],
+  "configPath": ["xpack", "apm"],
+  "extraPublicDirs": ["public/style/variables"],
   "requiredBundles": [
     "kibanaReact",
     "kibanaUtils",
     "observability",
-    "home"
+    "home",
+    "maps"
```

Comment:
Perhaps add a comment that this is used by csm. This way we can remove it when csm is moved to a separate app.

Reply:
JSON doesn't have comments.